### ISSUE TITLE
t/002-scalar.t fails with non-English locale (0.99_008)

### ISSUE THREAD

**Issue by eserte** [garu/Data-Printer#126]
With a German locale:
```
#   Failed test '$! is a dualvar'
#   at t/002-scalar.t line 215.
#          got: '"Datei oder Verzeichnis nicht gefunden" (dualvar: 2)'
#     expected: '"No such file or directory" (dualvar: 2)'
# Looks like you failed 1 test of 41.
t/002-scalar.t ........... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/41 subtests 
```
This happens possibly only with older perls (< 5.22).

**Comment by garu** [comment]
Thanks @eserte! Just fixed the test in 5c87621, will issue a new CPAN release pretty soon!

**Comment by eserte** [comment]
Confirmed, does not seem to happen anymore with _009
